### DISCUSSION TITLE
Tailwind 3 with CRA 5 and CRACO

### DISCUSSION THREAD

**Discussion by nickgieschen** [tailwindlabs/tailwindcss#11073] [Help]
I'm upgrading to Tailwind 3 and CRA, but am also using CRACO for features independent of tailwind. (In my case to use babel plugins.) That is, I know CRACO is not necessary for tailwind 3 anymore, but I need it for other things. Anyway, tailwind doesn't seem to get called during the build and I'm not sure why. Is it because I'm using CRACO? If so, is there a way to use CRACO with CRA and tailwind?

**Comment by zivnadel** [comment]
Did you try removing CRACO and building the app?

**Comment by nickgieschen** [comment]
No. I need CRACO for reasons other than tailwind.
